<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assets Loader - [BETA]</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbl_icon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources/resources.qrc">:/icons/box</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Assets Loader</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
        </font>
       </property>
       <property name="text">
        <string>Assets Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="path_line"/>
     </item>
     <item>
      <widget class="QToolButton" name="btn_browser">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QListWidget" name="list_widget">
     <property name="font">
      <font>
       <family>JetBrains Mono</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QScrollBar::sub-page:vertical {
background: none;
}

QScrollBar::add-page:vertical {
background: none;
}
QScrollBar:vertical {
     border: 2px solid grey;
     background: #B8C6D9;
     width: 15px;
     margin: 5px 0 5px 0;
 }
 QScrollBar::handle:vertical {
     background: #F2F2F2;
     min-height: 5px;
 }
 QScrollBar::add-line:vertical {
     border: 2px solid grey;
     background: #B8C6D9;
     height: 5px;
     subcontrol-position: bottom;
     subcontrol-origin: margin;
 }

 QScrollBar::sub-line:vertical {
     border: 2px solid grey;
     background: #B8C6D9;
     height: 5px;
     subcontrol-position: top;
     subcontrol-origin: margin;
 }
</string>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::ListMode</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_load">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
        </font>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_close">
       <property name="font">
        <font>
         <family>JetBrains Mono</family>
        </font>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
